setup.cfg

```diff
@@ -27,14 +27,14 @@ install_requires =
     nipype >= 1.5
     nitime
     nitransforms >= 20.0.0rc3,<20.2
-    niworkflows @ git+https://github.com/nipreps/niworkflows.git@master
+    niworkflows @ git+https://github.com/effigies/niworkflows.git@enh/multiapply-identity
```

Comment:
```suggestion
    niworkflows @ git+https://github.com/nipreps/niworkflows.git@master
```

Reply:
Should we pin to master or to the commit hash ?

Reply:
Checked and it looked like you're generally pinning to `master` rather than to a hash, so resolving my suggestion !

Reply:
We will release the dependencies and pin to those before releasing fmriprep, but this means we don't have to have 10 commits dedicated to updating hashes.